queries.sql

```diff
@@ -1,3 +1,46 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals
+WHERE name LIKE '%mon';
+
+SELECT * FROM animals
+WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-12-31';
+
+SELECT * FROM animals
+WHERE neutered = 't' and escape_attempts < 3;
+
+SELECT date_of_birth FROM animals
+WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals
+WHERE weight_kg > 10.5;
+
+SELECT * FROM animals
+WHERE neutered = 't';
+
+SELECT * FROM animals
+WHERE name <> 'Gabumon';
+
+SELECT * FROM animals
+WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+
+-- transactions
```

Comment:
- You have missed many required transaction,kindly add those.
- [ ] `UPDATE` the animals table by setting the species column to unspecified, then roll back the change.

```sql
   BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals;
```
- [ ] `UPDATE` the animals table by setting the species column to digimon for all animals that have a name ending in mon. & then `UPDATE` the animals table by setting the species column to pokemon for all animals that don't have species already set. & Commit the transaction.

```sql
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals;
```
- [ ] delete all records in the animals table, then roll back the transaction.